client/src/pages/Settings/Settings.tsx

```diff
@@ -17,7 +19,7 @@ const settingsMenu = [
   {
     name: 'Profile photo',
     to: '/profile/settings/profile-photo',
-    component: <SettingHeader header="Profile Photo" />,
+    component: <ProfilePhoto header="Profile Photo" imgPhoto="" />,
```

Comment:
The settings page automatically passes in the currentUser and currentProfile which will have the profile photo. I would like to keep this pretty basic and just accept header props for the settings page.

Reply:
but it will not display profile photo page, only header